src/troute-network/troute/HYFeaturesNetwork.py

```diff
@@ -414,7 +416,15 @@ def preprocess_network(self, flowpaths, nexus):
         # Location information will be used to advertise tailwater locations of diffusive domains 
         # to the model engine/coastal models
         self._nexus_latlon = nexus
-
+    def crosswalk_nex_flowpath_poi(self, flowpaths, nexus):
+        
+        mask_flowpaths = flowpaths['toid'].str.startswith('nex-')
```

Comment:
Can you add `toid`s that start with `tnex-` as well? We should capture these terminal nexus points as well.